1-exercises/A-reading-list/script.test.js

```diff
@@ -34,50 +34,50 @@ describe("Reading list", () => {
     const readingList = page.window.document.querySelector("#reading-list");
 
     expect(readingList).toHaveTextContent("The Design of Everyday Things");
-    expect(readingList).toHaveTextContent("Don Norman");
+    // expect(readingList).toHaveTextContent("Don Norman");
```

Comment:
If you comment out the tests your pipeline will pass, although it doesn't mean the solution is correct because there's no test which would check it's correct.

It's best not to touch the tests, they are there to help you.